docs/source/configure/index.rst

```diff
@@ -181,21 +184,80 @@ bin/irohad --genesis_block example/genesis.block --overwrite_ledger --verbosity=
 Windows
 -------
 
-``setx IROHA_BLOCK_STORE_PATH=/tmp/block_store/
-setx IROHA_TORII_PORT=50051
+``setx IROHA_TORII_PORT=50051
 setx IROHA_INTERNAL_PORT=10001
 .....
 ``
 
-Shell
------
+PowerShell
+----------
 
 ``$Env:IROHA_BLOCK_STORE_PATH=/tmp/block_store/
 $Env:IROHA_TORII_PORT=50051
 $Env:IROHA_INTERNAL_PORT=10001
 .....
 ``
 
+Parameter names
+---------------
+
+As you can see, the parameter names are not the same as in the configuration file.
+
+They are formed from the config structure, fixed label IROHA is added to the beginning and everything is uppercased and joined with _.
+Let us look a bit closer at how they are structured:
+
+**With simple string values**
+
+In config file:
+
+``"block_store_path" : "/tmp/block_store/" ``
+
+In env variables:
+
+``IROHA_BLOCK_STORE_PATH=/tmp/block_store/``
+
+
+**With arrays**
+
+Arrays are indexed starting with 0 and should be in direct order without skipping any numbers:
+
+In config file:
+
+``"initial_peers" : [{"address":"127.0.0.1:10001", "public_key": "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929"}, {"address":"127.0.0.1:10002", "public_key": "bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308920"}] ``
+
+In env variables:
+
+``IROHA_INITIAL_PEERS_0_ADDRESS=127.0.0.1:10001
+IROHA_INITIAL_PEERS_0_PUBLIC_KEY=bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308929
+IROHA_INITIAL_PEERS_1_ADDRESS=127.0.0.1:10002
+IROHA_INITIAL_PEERS_1_PUBLIC_KEY=bddd58404d1315e0eb27902c5d7c8eb0602c16238f005773df406bc191308920``
+
+**User-provided user dictionaries**
+
+User-provided dictionary keys are a bit trickier: the key and the value are set in separate vars
+
+In config file:
+
+``"crypto": {"providers": {
```

Comment:
maybe an example with loggers would be easier to grasp:

```
  "log": {
    "level": "info",
    "children": {
      "KeysManager": {
        "level": "trace"
      },
      "Irohad": {
        "level": "error"
      }
    }
  }
```

Reply:
There is an example below

Reply:
indeed